KsApi/models/BackingTests.swift

```diff
@@ -43,4 +44,15 @@ final class BackingTests: XCTestCase {
     XCTAssertEqual(1, backing.value?.sequence)
     XCTAssertEqual(Backing.Status.pledged, backing.value?.status)
   }
+
+  func testPledgeAmount() {
+    let backing = Backing.template
+      |> Backing.lens.reward .~ Reward.postcards
+      |> Backing.lens.rewardId .~ Reward.postcards.id
+      |> Backing.lens.shippingAmount .~ 116
```

Comment:
I'd really chose some easier numbers here for better understanding of what's going on in here 😄